src/store/index.ts

```diff
@@ -27,6 +27,8 @@ import transferReducer, { TransferState } from "src/store/transactions/transfer/
 import { TransferAction } from "src/store/transactions/transfer/transfer.actions";
 import { ExitAction } from "src/store/transactions/exit/exit.actions";
 import exitReducer, { ExitState } from "./transactions/exit/exit.reducer";
+import { WithdrawAction } from "./transactions/withdraw/withdraw.actions";
+import withdrawReducer, { WithdrawState } from "./transactions/withdraw/withdraw.reducer";
```

Comment:
We don't have a policy for imports as we don't have any rule in `eslint` but I'll add these as absolute to keep them in the same fashion of the others.

Reply:
I thought we had a non-written policy, especially because [you asked me to do this change in the past](https://github.com/hermeznetwork/wallet-ui/pull/641#discussion_r704337374).

I think we should avoid deeply nested imports as `import { foo } from "../../../../utils/currencies"` since they are difficult to read/write and are non-portable (no easy copy/paste). That said, I'm perfectly ok with relative imports as this one.

In any case, if we can enforce this rule with `eslint` it would be great!

Reply:
Yep, but my comment was to enforce the same style in the file, but I'm okay with both. The problem is the same with both approaches, so that's why I don't prefer one or another. Maybe if I need to choose an approach I will choose the relative, as we don't have too much nesting in our app and they are easy to write manually (you don't need to know all the path of a file to import it).

```js
import { foo } from "../../../../utils/currencies"
```

```js
import { foo } from "src/folder/folder/folder/utils/currencies"
```

Reply:
Whatever we decide, I think we should enforce the same style in the whole project.

I prefer the absolute path approach to be able to copy/paste and because, although it is true that you have to write them each time, you end up memorizing them and you have the help of the editor.